### PR TITLE
Update to Rocky Linux 9

### PR DESCRIPTION
**Pull request by khaledk2**
This PR contains the required modifications to work with Rokcy Linux 9

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -4,22 +4,22 @@
 # minio S3 gateway with custom users in front of an external S3 store
 # https://github.com/minio/minio/issues/8045
 
-- name: Install docker-py
+- name: Install docker
   become: true
-  yum:
-    name: docker-python
+  ansible.builtin.pip:
```
Comment (jburel) [inline tasks/main.yml:9]: yum packaging was not following the latest version of the library
installing via pip is different. Not possible to install using dnf on RockyLinux 9?

Comment (khaledk2) [reply tasks/main.yml:9]: I could not find a dnf installation package for it.

Comment (jburel) [reply tasks/main.yml:9]: this means that we should pin the current version to be sure it does not change every time someone runs the role

Comment (khaledk2) [reply tasks/main.yml:9]: I have added a new variable to control the docker version.

**Comment by jburel** [comment]
Thanks @khaledk2

**Pull request merged**